packages/components/src/popover/utils.js

```diff
@@ -4,6 +4,7 @@
  */
 const HEIGHT_OFFSET = 10; // used by the arrow and a bit of empty space
 const isMobileViewport = () => window.innerWidth < 782;
+const isRtl = () => document.documentElement.dir === 'rtl';
```

Comment:
RTL is an acronym, `isRTL` would be the correct naming.

https://github.com/WordPress/gutenberg/blob/master/docs/reference/coding-guidelines.md#abbreviations-and-acronyms